### PR TITLE
specify that `set_language` need syntect to work

### PR DESCRIPTION
**Pull request by sucrecacao**
## Summary
specify that `set_language` need syntect to work

## Related Issues
https://github.com/xi-editor/xi-editor/issues/1304

### CODE REVIEW

File: docs/docs/frontend-protocol.md [hunk 1]
```diff
@@ -736,7 +736,7 @@ Updates existing scope spans starting at offset `start` until offset `len`.
 
 ### update_annotations
 
-`update_annotations {"start": 0, "len": 20, "spans": [{ "start": 0, "end": 4, "data": null }], "annotation_type": "find", "rev": 3 }`
+`update_annotations {"start": 0, "len": 20, "spans": [{ "start": 0, "end": 4, "data": null }], "annotation_type": "find", "rev":/ 3 }`
```
Comment (cmyr) [inline docs/docs/frontend-protocol.md:739]: I assume this is a mistake?

**Review by cmyr — COMMENTED** [review]
Thanks, happy to take this patch with one fixup. :)

File: docs/docs/frontend-protocol.md (hunk 1 above)
Comment (sucrecacao) [reply docs/docs/frontend-protocol.md:739]: Yes my bad :s

Comment (cmyr) [inline docs/docs/frontend-protocol.md:739]:
```suggestion
`update_annotations {"start": 0, "len": 20, "spans": [{ "start": 0, "end": 4, "data": null }], "annotation_type": "find", "rev": 3 }`
```

**Pull request merged**